SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetrics.cpp

```diff
@@ -4,13 +4,46 @@
 
 #include "Engine/Engine.h"
 #include "EngineGlobals.h"
+#include "ProfilingDebugging/TraceAuxiliary.h"
 
 #include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Schema/ServerWorker.h"
 #include "SpatialGDKSettings.h"
 #include "Utils/SchemaUtils.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialMetrics);
 
+namespace SpatialMetricsPrivate
+{
+enum class EServerCommands : uint8
+{
+	StartInsights,
+
+	ServerCommandsCount,
+	ServerCommandInvalid = ServerCommandsCount,
+};
+
+const FString ServerCommandNames[static_cast<uint8>(EServerCommands::ServerCommandsCount) + 1] = { TEXT("StartInsights"), TEXT("Invalid") };
+
+const FString& ServerCommandsEnumToString(const EServerCommands Command)
+{
+	return ServerCommandNames[static_cast<uint8>(Command)];
+}
+
+const EServerCommands ServerCommandsStringToEnum(const FString& Command)
```

Comment:
This functionality can be provided by UENUM but would require ServerCommandInvalid to be it's own value I think

Reply:
Think I might be misremembering this..

Reply:
Ah you can do EnumToString but StringToEnum is just a loop/cmp over EnumToString(i) = String